src/qt/createwalletdialog.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2019 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_QT_CREATEWALLETDIALOG_H
+#define BITCOIN_QT_CREATEWALLETDIALOG_H
+
+#include <QDialog>
+
+class WalletModel;
+
+namespace Ui {
+    class CreateWalletDialog;
+}
+
+/** Dialog for creating wallets
+ */
+class CreateWalletDialog : public QDialog
+{
+    Q_OBJECT
+
+public:
+    explicit CreateWalletDialog(QWidget* parent);
+    virtual ~CreateWalletDialog();
+
+    QString walletName() const;
+    bool encrypt() const;
+    bool disablePrivateKeys() const;
+    bool blank() const;
```

Comment:
Nit: following my comment above, naming these `encryptWallet` and `makeBlankWallet` (like `disablePrivateKeys`) seems more precise/coherent/would improve git grepping.